### PR TITLE
Change MRP to SOR

### PR DESCRIPTION
**Pull request by mglidden**

### CODE REVIEW

File: _data/trip_fees.yml [hunk 1]
```diff
@@ -39,9 +39,13 @@
...
     category: trip
 -
-    name: "MITOC Rock Program $100"
+    name: "School of Rock Fee"
     price: 100
     category: trip
+-
+    name: "School of Rock T-Shirt"
+    price: 40
+    category: trip
```
Comment (mglidden) [inline _data/trip_fees.yml:48]: I know that this isn't a trip, but I don't see any way to do a set fee for a gear rental/purchase.

File: _data/trip_fees.yml [hunk 2]
```diff
@@ -39,9 +39,13 @@
     price: 10
     category: trip
 -
-    name: "MITOC Rock Program $100"
+    name: "School of Rock Fee"
```
Comment (DavidCain) [inline _data/trip_fees.yml:42]: So I would recommend including the fee directly in the name, to be consistent with all the other entries:

image

File: _data/trip_fees.yml (hunk 1 above)
Comment (DavidCain) [reply _data/trip_fees.yml:48]: Gear purchases are in `_data/gear_prices.yml`. That list populates https://mitoc.mit.edu/rentals/prices

For example, a MITOC shirt:
https://github.com/mitoc/website/blob/4c69b6fbf0ab2c2fb12ff7be3145af94204b142e/_data/gear_prices.yml#L212-L213

**Review by DavidCain — CHANGES_REQUESTED** [review]
I think you'll want to change a couple other places too to complete the name switch!

https://github.com/mitoc/website/blob/4c69b6fbf0ab2c2fb12ff7be3145af94204b142e/_pages/activities/rock-climbing.md#L18

https://github.com/mitoc/website/blob/4c69b6fbf0ab2c2fb12ff7be3145af94204b142e/_pages/events/school-of-rock.md#L2-L13

We should arguably keep the /events/school-of-rock link, but just have it redirect to /events/mitoc-rock-program or similar

**Comment by DavidCain** [comment]
@mglidden - I presume we still do want to complete the name change (back) from MRP to SoR? @ColeDCrawford just took care of removing the jumbo banner, and we have some other changes to trip fees in flight. You'll likely want to either rebase this PR against latest `main` or just open a new one.

All good to take care of that? I'm happy to try to apply the change myself too.

File: _data/trip_fees.yml (hunk 1 above)
Comment (mglidden) [reply _data/trip_fees.yml:48]: Yes, but that won't cause it to show up on the /pay page. We want to have it there to make it easy to track who has completed payment for their shirts.

**Pull request merged**